### PR TITLE
Activity selection

### PR DESCRIPTION
**Pull request by NEEHITGOYAL**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description
Added solution of activity selection problem 
with the use of Dynamic Array of pairs in c++

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#826

### CODE REVIEW

**Comment by NEEHITGOYAL** [comment]
kindly review and merge this PR as part of hakin codes

**Comment by NEEHITGOYAL** [comment]
KINDLY CHECK IT @HarshCasper

File: C-Plus-Plus/cp/Activity_Selection.cpp [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+12 15
+20 30
+
+Sample Output
+2
+
+Explanation
+Person can only do 0th and 2nd activities.
+**/
+#include<iostream>
```
Comment (siddharth25pandey) [inline C-Plus-Plus/cp/Activity_Selection.cpp:28]: Add single header file
**bits/stdc++**

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
Small modification is required
Else good to go!

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@NEEHITGOYAL please make the following changes:

1. I am seeing many changes in the `readMe.md` file, I am not sure if all these are required for this PR, if not please revert them.

2. Your code indentation seems to be a bit off, can you please fit it?

**Comment by NEEHITGOYAL** [comment]
@Kajol-Kumari due to so many days there were many pr merging and for resolving conflicts i have to make chages in readme and i still see there are two competitive coding links in readme i don't know who has done that i have made changes from my side and also idented the code you can re review

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@NEEHITGOYAL I can see so much edit in README file. Just add `- [ActivitySelection](cp/Activity_Selection.cpp)`.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@NEEHITGOYAL please use 
```
git fetch
git checkout upstream/master -- path/to/readme.md
```
this will revert all the changes and make the readme same as upstream/master(Here I  am assuming that you have added the original repo as upstream)

After doing this, just add your path file in readme, commit and push the changes.

For reference, do checkout [here](https://stackoverflow.com/questions/3334475/git-how-to-update-checkout-a-single-file-from-remote-origin-master)

**Comment by NEEHITGOYAL** [comment]
Thankyou @Kajol-Kumari

**Comment by NEEHITGOYAL** [comment]
i have updated readme same as it is now and added activity selection link and resolved all conflicts @Kajol-Kumari @iamrajiv  please check it and merge it thank you

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Review by iamrajiv — APPROVED** [review]

**Comment by NEEHITGOYAL** [comment]
@siddharth25pandey @Abhijit2505  kindly merge it

**Review by siddharth25pandey — APPROVED** [review]
@NEEHITGOYAL Good to go!
Able to merge
Thanks!

**Comment by siddharth25pandey** [comment]
@all-contributors please add @NEEHITGOYAL for his code.

**Pull request merged**